django_quickbooks/settings.py

```diff
@@ -75,9 +75,10 @@ def import_from_string(val, setting_name):
     """
     try:
         # Nod to tastypie's use of importlib.
-        module_path, class_name = val.rsplit('.', 1)
-        module = import_module(module_path)
-        return getattr(module, class_name)
+        if val:
```

Comment:
why we need check `val` for existence?

Reply:
I couldn't understand the reason, but while processing the response, it was throwing an error, because val was empty was empty sometimes